docs/src/usage.md

```diff
@@ -250,12 +295,26 @@ julia> diagram.Y
  [-100.0, 400.0]
 ```
 
-## Generating the influence diagram
+## Generating and solving the model
 
-The final part of modeling an influence diagram using the Decision Programming package is generating the full influence diagram. This is done using the `generate_diagram!` function.
+The final part is generating and solving the model. Generating model is done using the `generate_model` function.
 ```julia
-generate_diagram!(diagram)
+model, z, variables = generate_model(diagram, model_type="RJT")
 ```
-In this function, first, the probability and utility matrices in fields `X` and `Y` are sorted according to the chance and value nodes' indices.
+The execution of `generate_model` can be divided into six parts:
+
+1. Diagram generation using `generate_diagram!`. Generates the influence diagram. Probability and utility matrices are sorted in fields `X` and `Y` according to diagram.Names. Path probability and path utility types are initialized and added into fields `P` and `U`, respectively. By default `generate_model` and `generate_diagram!` use the default path probabilities and utilities, which are defined as the joint probability of all chance events in the diagram and the sum of utilities in value nodes, respectively.
```

Comment:
Add a subject to the second sentence. Now it sounds a bit funny

Reply:
Fixed